Examples/JamitFoundationExample/JamitFoundationExample.xcodeproj/project.pbxproj

```diff
@@ -853,7 +858,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = DB7EUU9D79;
 				INFOPLIST_FILE = App/SupportingFiles/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
See comment above for the deployment target. We need 9.0 for a specific app we are supporting.

Reply:
Okay, but this deployment target is actually **only** for the example project. Therefore it should be irrelevant for any app which uses JamitFoundation (the JamitFoundation deployment target didn't change).